modules/imaging_uploader/jsx/UploadForm.js

```diff
@@ -341,20 +370,23 @@ class UploadForm extends Component {
       }
     } else if (xhr.status == 0) {
       errorMessage = {
-        'mriFile': ['Upload failed: a network error occured'],
+        'mriFile': [t('Upload failed: a network error occured',
+          {ns: 'imaging_uploader'})],
       };
     } else if (xhr.status == 413) {
       errorMessage = {
         'mriFile': [
-          'Please make sure files are not larger than '
-          + this.props.maxUploadSize,
+          t('Please make sure files are not larger than',
```

Comment:
Different languages can the variable in different places grammatically.

This needs to be a template string that includes a variable rather than using string concatenation:

It would be something like:
`t('Please make sure files are not larger than {maxFileSize}', { ns: 'imaging_uploader', maxFileSize: this.props.maxUploadSize})`

(untested.. can't remember off the top of my head if it needs the variable delimited by `{varname}` or `{{varname}}`)